src/Rasterization/Renderers/SVGRectRenderer.php

```diff
@@ -86,4 +156,76 @@ protected function renderStroke($image, array $params, $color, $strokeWidth)
             $color
         );
     }
+
+    private function renderStrokeRounded($image, array $params, $color, $strokeWidth) {
+        $x1 = $params['x1'];
+        $y1 = $params['y1'];
+        $x2 = $params['x2'];
+        $y2 = $params['y2'];
+        $rx = $params['rx'];
+        $ry = $params['ry'];
+
+        $halfStrokeFloor = floor($strokeWidth / 2);
+        $halfStrokeCeil  = ceil($strokeWidth / 2);
+
+        // top
+        imagefilledrectangle(
+            $image,
+            $x1 - $halfStrokeFloor + $rx*1.5,  $y1 - $halfStrokeFloor,
+            $x2 + $halfStrokeFloor - $rx*1.5,  $y1 + $halfStrokeCeil - 1,
+            $color
+        );
+        // bottom
+        imagefilledrectangle(
+            $image,
+            $x1 - $halfStrokeFloor + $rx*1.5,  $y2 - $halfStrokeCeil + 1,
+            $x2 + $halfStrokeFloor - $rx*1.5,  $y2 + $halfStrokeFloor,
+            $color
+        );
+        // left
+        imagefilledrectangle(
+            $image,
+            $x1 - $halfStrokeFloor,     $y1 + $halfStrokeCeil + $ry/2,
+            $x1 + $halfStrokeCeil - 1,  $y2 - $halfStrokeCeil - $ry/2,
+            $color
+        );
+        // right
+        imagefilledrectangle(
+            $image,
+            $x2 - $halfStrokeCeil + 1,  $y1 + $halfStrokeCeil + $ry/2,
+            $x2 + $halfStrokeFloor,     $y2 - $halfStrokeCeil - $ry/2,
+            $color
+        );
```

Comment:
Why draw filled rectangles instead of using `imagesetthickness()` followed by drawing lines?